packages/algob/src/lib/pyteal.ts

```diff
@@ -0,0 +1,36 @@
+import { spawnSync, SpawnSyncReturns } from "child_process";
+import path from "path";
+
+import { ASSETS_DIR } from "../internal/core/project-structure";
+
+const pythonExt = ".py";
```

Comment:
Above file uses `pyExt`. Let's keep the same here as well.

Reply:
this file is only for task purpose , i don't see any use of this, this file will be removed,(all code is moved to PyCompileOp)